Makefile

```
@@ -0,0 +1,177 @@
+	else \
+		./scripts/docker-run.sh gnosis logs -f; \
+	fi
+
+# NuGet package operations
+pack:
+	./scripts/nuget-pack.sh
+
+pack-clean:
+	./scripts/nuget-pack.sh --clean
+
+push:
```

checkmake (0.2.2)

[warning] Target body for "push" exceeds allowed length of 5 (16).

(maxbodylength)